sdcm/nemesis.py

```diff
@@ -6660,3 +6711,21 @@ class EndOfQuotaNemesis(Nemesis):
 
     def disrupt(self):
         self.disrupt_end_of_quota_nemesis()
+
+
+class GrowShrinkZeroTokenNode(Nemesis):
```

Comment:
I think we need another nemesis flag like 'zero_token' to mark these nemesis that work with this feature, so we could exclude/include them in tests easily. @roydahan WDYT?

Reply:
added